rebar.config.erl

```diff
@@ -235,7 +247,8 @@ relx_overlay(ReleaseType) ->
 etc_overlay(ReleaseType) ->
     PluginApps = relx_plugin_apps(ReleaseType),
     Templates = emqx_etc_overlay(ReleaseType) ++
-                lists:append([plugin_etc_overlays(App) || App <- PluginApps]),
+                lists:append([plugin_etc_overlays(App) || App <- PluginApps]) ++
```

Comment:
It should work if we just extend `PluginApps` with community app names ?

Reply:
No, `plugin_etc_overlays/1` calls `find_conf_files/1`, which looks for the corresponding conf file. Since community plugins are not there when rendering the rebar.config, it fails to fetch the name of `.conf`.

Reply:
ah, right, now I remembered.
then we should add them hard-coded (by assuming each plugin app (pulled as rebar dependency) should have a `.conf` file

```
%% NOTE: for apps fetched as rebar dependency (there is so far no such an app)
%% the overlay should be hand-coded but not to rely on build-time wildcards.
```

This is because if we look in `_build` dir then the generated config is non-deterministic.

Reply:
That's right. fixed to

```
community_plugin_etc_overlays(App0) ->
    App = atom_to_list(App0),
    {"{{base_dir}}/lib/"++ App ++"/etc/" ++ App ++ ".conf", "etc/plugins/" ++ App ++ ".conf"}.
```